cstrml/staking/src/lib.rs

```diff
@@ -2083,7 +2077,8 @@ impl<T: Config> Module<T> {
                 total_payout = total_payout.saturating_sub(used_fee);
 
                 // 5. Split the payout for staking and authoring
-                let total_authoring_payout = T::AuthoringAndStakingRatio::get() * total_payout;
+                let num_of_validators = points.individual.len();
```

Comment:
Using `CurrentElected` should be more accurate.

Reply:
Done